demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1113,3 +1116,78 @@ def is_context_change_in_readme(self) -> bool:
                     valid = False
 
         return valid
+
+    def is_there_separators_in_names(self) -> bool:
+        """
+        Check if there are separators in the integration folder or files.
+
+        Returns:
+            true if the folder/files names are valid and there are no separators, and false if not.
+        """
+        is_unified_integration = self.current_file.get('script', {}).get('script', '') not in ['-', '']
+
+        if is_unified_integration:
+            return True
+
+        answers = [
+            self.check_separators_in_folder(),
+            self.check_separators_in_files()
+        ]
+
+        return all(answers)
+
+    def check_separators_in_folder(self) -> bool:
+        """
+        Check if there are separators in the integration folder.
+
+        Returns:
+            true if the name is valid and there are no separators, and false if not.
+        """
+
+        integration_folder_name = self.file_path.rsplit('/', 2)[1]
```

Comment:
```suggestion
        integration_folder_name = os.path.basename(os.path.dirname(self.file_path))
```
this will make it work on windows as well